### PR TITLE
Fixed Project Creation issue if OutputDirectory is specified for hybr…

### PR DESCRIPTION
**Pull request by trooper2013**
…id apps

### CODE REVIEW

File: node/forceios.js [hunk 1]
```diff
@@ -208,11 +211,11 @@ function buildArgsFromArgMap(config) {
 // -----
 
 function createArgProcessorList() {
-    
+
     var argProcessorList = new commandLineUtils.ArgProcessorList();
 
     // App type
-    addProcessorFor(argProcessorList, 'apptype', 'Enter your application type (native, native_swift, react_native, hybrid_remote, or hybrid_local):', 'App type must be native, native_swift, react_native, hybrid_remote, or hybrid_local.', 
+    addProcessorFor(argProcessorList, 'apptype', 'Enter your application type (native, native_swift, react_native, hybrid_remote, or hybrid_local):', 'App type must be native, native_swift, react_native, hybrid_remote, or hybrid_local.',
```
Comment (kchitalia) [inline node/forceios.js]: What was changed here?

File: node/forceios.js [hunk 2]
```diff
@@ -228,7 +231,7 @@ function createArgProcessorList() {
     addProcessorFor(argProcessorList, 'organization', 'Enter your organization name (Acme, Inc.):', 'Invalid value for organization: \'$val\'.',  /\S+/);
 
     // Start page
-    addProcessorFor(argProcessorList, 'startpage', 'Enter the start page for your app (only applicable for hybrid_remote apps):', 'Invalid value for start page: \'$val\'.', /\S+/, 
+    addProcessorFor(argProcessorList, 'startpage', 'Enter the start page for your app (only applicable for hybrid_remote apps):', 'Invalid value for start page: \'$val\'.', /\S+/,
```
Comment (kchitalia) [inline node/forceios.js]: What was changed here?

File: node/forceios.js [hunk 3]
```diff
@@ -249,7 +252,7 @@ function createArgProcessorList() {
 // * validation: function or regexp or null (no validation)
 // * preprocessor: function or null
 // * postprocessor: function or null
-// 
+//
```
Comment (kchitalia) [inline node/forceios.js]: Same here.

File: node/forceios.js [hunk 4]
```diff
@@ -47,7 +47,7 @@ function main(args) {
         process.exit(0);
         break;
     case 'create':
-        processorList = createArgProcessorList(); 
+        processorList = createArgProcessorList();
```
Comment (kchitalia) [inline node/forceios.js]: Looks like nothing changed in this line.

Comment (bhariharan) [inline node/forceios.js]: Removed additional whitespace at EOL.

File: node/forceios.js (hunk 1 above)
Comment (bhariharan) [inline node/forceios.js]: Additional whitespace at EOL was removed I'm guessing.

File: node/forceios.js (hunk 2 above)
Comment (bhariharan) [inline node/forceios.js]: Same here.

**Comment by bhariharan** [comment]
Nice! 👍

**Pull request merged**